### PR TITLE
Release 0.1.0

### PR DESCRIPTION
**Pull request by jmagnuson**

### CODE REVIEW

File: LICENSE-MIT [hunk 1]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2017 Jon Magnuson
```
Comment (nastevens) [inline LICENSE-MIT:3]: 2017 -> 2020

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 # libevent-rs
+[![](https://meritbadge.herokuapp.com/libevent)](https://crates.io/crates/libevent)
```
Comment (nastevens) [inline README.md:2]: I've never seen meritbadge, I've always used shields.io. Are there any pros/cons to one vs the other?

File: README.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
...
+[![Released API docs](https://docs.rs/libevent/badge.svg)](https://docs.rs/libevent)
```
Comment (nastevens) [inline README.md:3]: Another useful badge is the MSRV: Minimum rustc version

File: README.md (hunk 2 above)
Comment (jmagnuson) [reply README.md:2]: I just copy-pasta'ed from the `libevent-sys` README for ~laziness~ consistency. And looking at my other crates, some use shields.io while others use meritbadge, so there's really no distinct reasoning behind the choice. Probably good just to pick one and stick with it though 🤷

File: README.md (hunk 3 above)
Comment (jmagnuson) [reply README.md:3]: Made #12 to track this. As a sidenote, while looking for authoritative references for this I found https://github.com/rust-lang/rust/issues/65262 which seeks to allow declaring MSRV directly in the `Cargo.toml`, pretty cool.

**Pull request merged**